test/mock_client.ts

```diff
@@ -59,7 +78,5 @@ class MockClient {
     }
 
     // eslint-disable-next-line class-methods-use-this
-    async bulk(data) { return data; }
+    async bulk(data: any) { return data; }
```

Comment:
Since we are mocking the elasticsearch client, ee should validate the bulk request here (like elasticsearch would).